faust/stores/rocksdb.py

```diff
@@ -116,25 +126,57 @@ def __init__(
 
     def open(self, path: Path, *, read_only: bool = False) -> DB:
         """Open RocksDB database using this configuration."""
-        return rocksdb.DB(str(path), self.as_options(), read_only=read_only)
+        if ROCKSDICT_INSTALLED:
+            db_options = self.as_options()
+            db_options.set_db_paths(
+                [rocksdict.DBPath(str(path), self.target_file_size_base)]
+            )
+            db = DB(str(path), options=self.as_options())
+            db.set_read_options(rocksdict.ReadOptions(raw_mode=True))
+            return db
+        else:
+            return rocksdb.DB(str(path), self.as_options(), read_only=read_only)
```

Comment:
Should we reset `ROCKSDICT_INSTALLED = False` here?

Reply:
I think it's best we set that variable based on its import status:
https://github.com/faust-streaming/faust/pull/470/files#diff-083670c0eb5020882953ec86588aa93894a9c8d1371d5806e1c2da38807b5382R66-R75

However, if we add an option for users to selectively switch between the two, then that option would change the state of `ROCKSDICT_INSTALLED`.

Reply:
Took care of the feature in #470, let's continue there.